webapp/channels/src/components/channel_header_dropdown/menu_items/toggle_mute_channel/toggle_mute_channel.tsx

```diff
@@ -43,42 +43,25 @@ type Props = {
     actions: Actions;
 };
 
-export default class MenuItemToggleMuteChannel extends React.PureComponent<Props> {
-    handleClick = () => {
-        const {
-            user,
-            channel,
-            isMuted,
-            actions: {
-                updateChannelNotifyProps,
-            },
-        } = this.props;
-
-        updateChannelNotifyProps(user.id, channel.id, {
+export default function MenuItemToggleMuteChannel({id, isMuted, channel, user, actions}: Props) {
```

Comment:
Can we put each prop in a different line?
```suggestion
export default function MenuItemToggleMuteChannel({
    id,
    isMuted,
    channel,
    user,
    actions
}: Props) {
```